### PR TITLE
post for the 0.36 release

### PR DESCRIPTION
**Pull request by parkouss**

### CODE REVIEW

File: _posts/2015-04-17-0.36-release.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title:  "Release 0.36"
+author: Julien Pagès
+---
+
+This release includes a bugfix for the download in backgound feature introduced
+in 0.35 (see [bug 1153801]).
+
+The fennec profile is also taken in account now, meaning:
```
Comment (wlach) [inline _posts/2015-04-17-0.36-release.md]: A few changes I'd make:

The fennec profile is also taken in account now, meaning: -> Also, on Firefox for Android we now use a custom profile, meaning:

(Firefox for Android means something to more people than Fennec)

File: _posts/2015-04-17-0.36-release.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+- the profile on the device is no more overriden by mozregression
+- the profile sent by mozregression can be customized
+
+See [bug 1147576] for more information about it.
```
Comment (wlach) [inline _posts/2015-04-17-0.36-release.md]: for more information about it -> for more information

**Comment by wlach** [comment]
Looks good. Some minor suggestions on wording, but feel free to merge whenever.

**Comment by parkouss** [comment]
Thanks Will. :)

**Pull request merged**